### PR TITLE
[python] Add CVE-2015-3908

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@samuzzal-choudhury PTAL, thanks :)

**Comment by samuzzal-choudhury** [comment]
@msrb PTAL

File: database/python/2015/3908.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+references:
+    - http://lists.opensuse.org/opensuse-updates/2015-07/msg00051.html
+    - http://lists.opensuse.org/opensuse-updates/2015-08/msg00029.html
+    - http://www.ansible.com/security
+    - http://www.openwall.com/lists/oss-security/2015/07/14/4
+
+affected:
+    - name: ansible
+      version:
+        - "<1.9.2"
```
Comment (msrb) [inline database/python/2015/3908.yaml]: Unfortunately this syntax is not supported by VictimsDB: https://github.com/victims/victims-cve-db#version-string-common

You need to use `"<=1.9.1"`

Comment (msrb) [reply database/python/2015/3908.yaml]: @samuzzal-choudhury PTAL :wink:

**Pull request closed without merging**